Formula/flagd.rb

```diff
@@ -0,0 +1,45 @@
+class Flagd < Formula
+  desc "Feature flag daemon with a Unix philosophy"
+  homepage "https://github.com/open-feature/flagd"
+  url "https://github.com/open-feature/flagd.git", using: :git, tag: "v0.3.0", revision: "2ad206e635cc5b68ae42287b740ddfc86b0194ff"
+  license "Apache-2.0"
+  head "https://github.com/open-feature/flagd.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPRIVATE"] = "buf.build/gen/go"
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+      -X main.commit=#{Utils.git_head}
+      -X main.date=#{time.iso8601}
+    ]
```

Comment:
@SMillerDev @chenrui333 I added this as well since it was missing from the initial implementation